src/renderer/components/wallet/assets/AssetsTableCollapsable.tsx

```diff
@@ -275,10 +276,16 @@ export const AssetsTableCollapsable: React.FC<Props> = (props): JSX.Element => {
   useEffect(() => {
     // don't change openPanelKeys if user has already changed panel state
     if (!collapseChangedByUser) {
-      const keys: string[] = []
-      chainBalances.forEach((chainBalance, index) => {
-        if (RD.isSuccess(chainBalance.balances) && chainBalance.balances.value.length) keys.push(index.toString())
-      })
+      const keys = FP.pipe(
```

Comment:
Pls add a small note above `const keys`, something like `filter out empty list of balances and zero balances.